cla_backend/apps/call_centre/admin/forms.py

```diff
@@ -1,13 +1,22 @@
 from django import forms
-
```

Comment:
I'd prefer to keep this space between the third-party package block and the local package block